pandas/tests/io/formats/test_to_html.py

```diff
@@ -896,3 +896,12 @@ def test_to_html_float_format_object_col(datapath):
     result = df.to_html(float_format=lambda x: f"{x:,.0f}")
     expected = expected_html(datapath, "gh40024_expected_output")
     assert result == expected
+
+
+def test_to_html_multiindex_col_with_colspace(datapath):
+    # GH#53885
+    df = DataFrame([[1, 2], [3, 4]])
+    df.columns = MultiIndex.from_tuples([(1, 1), (2, 1)])
+    result = df.to_html(col_space=100)
+    expected = expected_html(datapath, "gh53885_expected_output")
```

Comment:
@mroeschke did you mean to replace this line with below 
```
expected = """
        <table border="1" class="dataframe">
        <thead>
            <tr>
            <th style="min-width: 100px;"></th>
            <th style="min-width: 100px;">1</th>
            <th style="min-width: 100px;">2</th>
            </tr>
            <tr>
            <th style="min-width: 100px;"></th>
            <th style="min-width: 100px;">1</th>
            <th style="min-width: 100px;">1</th>
            </tr>
        </thead>
        <tbody>
            <tr>
            <th>0</th>
            <td>1</td>
            <td>2</td>
            </tr>
            <tr>
            <th>1</th>
            <td>3</td>
            <td>4</td>
            </tr>
        </tbody>
        </table>
    """
```

This might make the test code larger / slightly less readable.

Reply:
Yes, this is more performant though than file reading

Reply:
ok got it, will make this change.